src/sdc11073/provider/scopesfactory.py

```diff
@@ -1,10 +1,33 @@
 """The module implements the function mk_scopes."""
-from urllib.parse import quote_plus
 
-from sdc11073.location import SdcLocation
+import urllib.parse
+
 from sdc11073.mdib.mdibprotocol import ProviderMdibProtocol
 from sdc11073.xml_types.wsd_types import ScopesType
 
+KEY_PURPOSE_SERVICE_PROVIDER = 'sdc.mds.pkp:1.2.840.10004.20701.1.1'
+BICEPS_URI_UNK = 'biceps.uri.unk'
```

Comment:
maybe provide information where this is from

Reply:
done